public/pages/Policies/components/PolicyEmptyPrompt/__snapshots__/PolicyEmptyPrompt.test.tsx.snap

```diff
@@ -43,3 +43,47 @@ exports[`<PolicyEmptyPrompt /> spec renders the component 1`] = `
   </a>
 </div>
 `;
+
```

Comment:
Not sure what happened here, seems like this duplicated above?

Reply:
Cleaned up